test/tools/isoltest.cpp

```diff
@@ -326,11 +331,13 @@ int main(int argc, char *argv[])
 		TestTool::editor = "/usr/bin/editor";
 
 	fs::path testPath;
+	string ipcPath;
+	bool disableIPC = false;
 	bool disableSMT = false;
 	bool formatted = true;
 	po::options_description options(
 		R"(isoltest, tool for interactively managing test contracts.
-Usage: isoltest [Options] --testpath path
+Usage: isoltest [Options] --testpath path --ipcpath ipcpath
```

Comment:
```suggestion
Usage: isoltest [Options] --testpath path {--ipcpath ipcpath, --no-ipc}
```
Or ``[--ipcpath ipcpath]``in "optional brackets"?

Or just two lines (with or without repeating ``Usage: ``):
```
Usage: isoltest [Options] --testpath path --ipcpath ipcpath
Usage: isoltest [Options] --testpath path --no-ipc
```

Reply:
I think we could follow the unix standard here and print:
```
Usage: isoltest [--testpath testpath] [--ipcpath ipcpath] [--no-ipc]
``` 
or 
```
Usage: isoltest [--no-ipc] [--testpath testpath] [--ipcpath ipcpath]
```

Reply:
Actually if we start guessing the ipcpath, then none of the options are particularly special and we could just leave it at ``Usage: isoltest [Options]``. In any case we can remove the ``--testpath testpath`` part here - I think it's only in there as a relic from the time where it wasn't optional.

Reply:
I've gone with `Usage: isoltest [Options]  --ipcpath ipcpath` for now, since we're not guessing it, yet.

Reply:
I think that "guessing" code needs to be shared with `soltest`, @Marenz what's your take on this?

Reply:
Saw this too late. But yeah, if they share command line parameters the defaults / handling should be shared